Lib/test/test_unpack_ex.py

```diff
@@ -308,12 +308,12 @@
     >>> a, *b, c, *d, e = range(10) # doctest:+ELLIPSIS
     Traceback (most recent call last):
       ...
-    SyntaxError: two starred expressions in assignment
+    SyntaxError: more than one starred expressions in assignment
 
     >>> [*b, *c] = range(10) # doctest:+ELLIPSIS
     Traceback (most recent call last):
       ...
-    SyntaxError: two starred expressions in assignment
+    SyntaxError: more than one starred expressions in assignment
```

Comment:
Can you add a test case for multiple star assignments like the example in the bpo?